### PR TITLE
Memcached-mutilate hmem workload

### PR DESCRIPTION
**Pull request by Damenus**

### CODE REVIEW

File: examples/kubernetes/workloads/memcached-mutilate/medium-wss/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  app: memcached-mutilate-medium-wss
+
+configMapGenerator:
+- name: mutilate
+  behavior: merge
+  literals:
+  - threads=6
+  - load_records=35150000 # 35.1M
+  - records=35150000      # 35.1M
+  - value=1000            # keys 30B + value 1000B * 35.1M <= 40GB
```
Comment (felidadae) [inline examples/kubernetes/workloads/memcached-mutilate/medium-wss/kustomization.yaml:19]: lacking paranthesis

File: examples/kubernetes/workloads/memcached-mutilate/_hmem/pmem/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- ../../medium
+- ../../medium-wss
+- ../../big
+- ../../big-wss
+
+
+commonAnnotations:
+  cri-resource-manager.intel.com/memory-type: |
+    service: pmem
+    generator: dram
```
Comment (ppalucki) [inline examples/kubernetes/workloads/memcached-mutilate/_hmem/pmem/kustomization.yaml:18]: there is bug here, I messed up, and it should be "generator: pmem", Marta is going to fix that for redis and you should for memcached

**Review by ppalucki — CHANGES_REQUESTED** [review]

File: examples/kubernetes/workloads/memcached-mutilate/_hmem/pmem/kustomization.yaml (hunk 2 above)
Comment (Damenus) [reply examples/kubernetes/workloads/memcached-mutilate/_hmem/pmem/kustomization.yaml:18]: Fixed

File: examples/kubernetes/workloads/memcached-mutilate/medium-wss/kustomization.yaml (hunk 1 above)
Comment (Damenus) [reply examples/kubernetes/workloads/memcached-mutilate/medium-wss/kustomization.yaml:19]: Fixed

**Review by ppalucki — APPROVED** [review]

**Pull request merged**